k8s/wordpress/README.md

```diff
@@ -8,6 +8,22 @@ WordPress is web publishing software used to create websites and blogs.
 
 Popular open stacks on Kubernetes packaged by Google.
 
+## Design
+
+![Architecture diagram](resources/wordpress-k8s-app-architecture.png)
```

Comment:
Please replace dashes with underscores in Secrets, to match the names from `Secrets` resource.

Reply:
corrected